nixos/modules/tasks/network-interfaces.nix

```diff
@@ -68,6 +68,16 @@ let
             bits in the prefix (<literal>${if v == 4 then "24" else "64"}</literal>).
           '';
         };
+
+        broadcast = mkOption {
+          type = types.str;
+          default = "+";
+          description = ''
+            Broadcast address of the interface. Either an IPv${toString v} address or "+" or "-",
+            which set or clear the bits of address after prefixLength,
+            yielding the highest or lowest address in the specified network.
```

Comment:
Can you make an example for `+` and `-` like 192.168.0.0/24 -> 192.168.0.255 for better understanding.

Reply:
Yeah that makes sense, added. On second thought, perhaps the mechanics of + and - ought to be ignored and just saying that they give the highest and lowest address in the network and the examples would be less confusing.

Reply:
I think it could be written as:

```
Either an IPv${toString v} address or <literal>+</literal> or <literal>-</literal>. <literal>+</literal> will yield the highest address in the subnet, while <literal>-</literal> the lowest.
```

followed by the example.